org.pipoypipagames.cowsrevenge.json

```diff
@@ -0,0 +1,29 @@
+{
+  "app-id": "org.pipoypipagames.cowsrevenge",
+  "branch": "stable",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "cowsrevenge",
+  "finish-args": [
+     "--socket=x11",
```

Comment:
This also needs --share=ipc in order to get x11 to work at top performance. http://docs.flatpak.org/en/latest/sandbox-permissions.html

Reply:
Didn't know that, thanks